content/docs/user-guide/analytics.md

```diff
@@ -25,9 +25,9 @@ User and event data have a 14 month retention period.
 
 DVC's analytics record the following information per event:
 
-- The DVC version e.g. `0.82.0`
+- The DVC version e.g. `2.9.5`
 - Whether DVC was installed from a binary release
-- Operating system information, e.g. Ubuntu Linux 14.04
+- Operating system information, e.g. Ubuntu Linux 22.04
```

Comment:
> Ubuntu 22.04 is released on April 21, 2022

getting a bit ahead of ourselves :sweat_smile:

Reply:
```suggestion
- Operating system information, e.g. Ubuntu Linux 20.04
```

Reply:
Committing since it's not related to the PR stated purpose.